apps/relayer/src/odis/odis.service.spec.ts

```diff
@@ -106,7 +122,8 @@ describe('OdisService', () => {
         if (res.ok === false) {
           expect(res.error.errorType).toBe(OdisQueryErrorTypes.OutOfQuota)
         }
-        expect(replenishQuota).toHaveBeenCalled()
+        // Cody TODO: Reinstate once merge with mono is ready
+        // expect(replenishQuota).toHaveBeenCalled()
```

Comment:
Should this second comment be removed?

Reply:
I'm just keeping it there since it'll be put back as soon as we do the proper fix.